app/models/category.rb

```diff
@@ -15,4 +15,8 @@ def to_json
   def self.to_json
     includes(:items).order(:description).all.map(&:to_json).to_json
   end
+
+  def value
+    items.all.map(&:value).inject(:+)
```

Comment:
They look to be runnning the same queries to me....
```
2.2.4 :009 >   Category.first.items.sum(:value)
  Category Load (0.4ms)  SELECT  "categories".* FROM "categories"  ORDER BY upper(description) LIMIT 1
   (0.2ms)  SELECT SUM("items"."value") FROM "items" WHERE "items"."category_id" = $1  [["category_id", 1]]
 => #<BigDecimal:7fdd8a942d88,'0.584E2',18(18)>
2.2.4 :010 > Category.first.items.all.map(&:value).inject(:+)
  Category Load (0.6ms)  SELECT  "categories".* FROM "categories"  ORDER BY upper(description) LIMIT 1
  Item Load (0.2ms)  SELECT "items".* FROM "items" WHERE "items"."category_id" = $1  [["category_id", 1]]
 => #<BigDecimal:7fdd8a8e3720,'0.584E2',18(36)>
2.2.4 :011 >
```
but yours is easier to read so i'll implement it.